Projects/ROS2NavigationExample/Registry/bootstrap.setreg

```diff
@@ -0,0 +1,10 @@
+{
+    "Amazon": {
+        "AzCore": {
+            "Bootstrap": {
+		"project_path": ".."
+            }
+        }
+    }
+}
+
```

Comment:
This file should be deleted.
Setting the `project_path` should not be done in the shared settings